### PR TITLE
improving navbar 

### PR DESCRIPTION
**Pull request by kumarishalini6**
#173 

There was so much stuff in the navbar I have removed  it and create a covid file from there links of patients and tests.
lifeMEDeasy - Personal - Microsoft​ Edge 5_26_2021 10_35_11 PM

### CODE REVIEW

File: src/components/NavbarItem.js [hunk 1]
```diff
@@ -1,3 +1,37 @@
...
+import React from 'react';
+import Navbar from 'react-bootstrap/Navbar';
+import Nav from 'react-bootstrap/Nav' ;
+import Container from 'react-bootstrap/Container';
+
+export default function NavbarItem() {
+    return (
+        <div>
+         <Navbar collapseOnSelect expand="lg" bg="dark" variant="dark">
+  <Container>
```
Comment (chetan-rns) [inline src/components/NavbarItem.js:11]: Is this commented file required?

File: src/components/NavbarItem.js [hunk 2]
```diff
@@ -49,17 +83,14 @@ class NavbarItem extends Component {
...
-                                        <NavLink href="/covidpatient" className="links" ><span className="fa fa-info fa-lg"></span> Covid-19 Patients  </NavLink>
-                                    </NavItem>
-                                    <NavItem active>
-                                        <NavLink href="/covidtest" className="links" ><span className="fa fa-info fa-lg"></span> Covid-19 Tests  </NavLink>
+                                        <NavLink href="/covid" className="links" ><span className="fa fa-info fa-lg"></span> Covid-19 </NavLink>
                                     </NavItem>
                                     <NavItem active>
 
-                                        <NavLink href="/emergency"  ><span className="fa fa-info fa-lg"></span> Emergencies</NavLink>
+                                        <NavLink href="/emergency" className="links" ><span className="fa fa-info fa-lg"></span> Emergencies</NavLink>
```
Comment (chetan-rns) [inline src/components/NavbarItem.js:90]: Clicking on emergency is throwing an error. Could you please take a look?

**Review by chetan-rns — CHANGES_REQUESTED** [review]

File: src/components/NavbarItem.js (hunk 2 above)
Comment (kumarishalini6) [reply src/components/NavbarItem.js:90]: ok let me check

File: src/components/NavbarItem.js (hunk 1 above)
Comment (kumarishalini6) [reply src/components/NavbarItem.js:11]: ok let me check

Comment (kumarishalini6) [reply src/components/NavbarItem.js:11]: sorry , no not required i was just checking something that's why

File: src/components/NavbarItem.js (hunk 2 above)
Comment (kumarishalini6) [reply src/components/NavbarItem.js:90]: can you please show me the error

Comment (kumarishalini6) [reply src/components/NavbarItem.js:90]: I have added className ="links" because of css

**Review by avinashkranjan — APPROVED** [review]
LGTM..💯

**Pull request merged**